src/imp_pl.rs

```diff
@@ -87,7 +87,7 @@ impl<T> OnceCell<T> {
         }
     }
 
-    pub fn get_or_try_init<F: FnOnce() -> Result<T, E>, E>(&self, f: F) -> Result<&T, E> {
+    pub(crate) fn get_or_try_init<F: FnOnce() -> Result<T, E>, E>(&self, f: F) -> Result<&T, E> {
```

Comment:
both `get_or_try_init` and `into_inner` should be just `pub` to be visible from outside

Reply:
Ah, sorry, that's pl impl, it should be `pub(crate)` indeed, thanks!